.github/workflows/buildAndDeploy.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
       - name: Tail Otel mock events for 60 seconds
         run: |
-          kubectl delete pods -l component=node-exporter -n monitoring
+          kubectl run dummy --image ubuntu -- /bin/bash -ec "while :; do echo '.'; sleep 5 ; done"
```

Comment:
Does it need to download the ubuntu image? Or is it already cached from some previous steps?